src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -4,17 +4,21 @@ import DrillDownTable from '@onaio/drill-down-table';
 import reducerRegistry from '@onaio/redux-reducer-registry';
 import superset from '@onaio/superset-connector';
 import { Dictionary } from '@onaio/utils';
+import _ from 'lodash';
+import { trimStart } from 'lodash';
```

Comment:
@kelvin-muchiri should grab what we need from lodash instead of bringing in the whole library?

Reply:
@cKellyDesign This is actually now unused. Seems like the pre-commit hook ignored it maybe cause its a dash?